packages/prettier-plugin-java/src/printers/packages-and-modules.js

```diff
@@ -14,7 +14,9 @@ const {
 
 class PackagesAndModulesPrettierVisitor {
   compilationUnit(ctx) {
-    return this.visitSingle(ctx);
+    const EOF = ctx.EOF[0];
+    delete ctx.EOF;
```

Comment:
mutating the CTX does not make sense.
The next person who tries to visit the CST won't get the same results! 😢

Reply:
You are totally right, I was getting an error of visitSingle since I added EOF. I could have just retrieve the children and call the visit on it.